.gitignore

```diff
@@ -1,4 +1,4 @@
-/node_modules
+node_modules
```

Comment:
Some tests have a `node_modules` folder to test module resolution but `jest` doesn't run them and I'm not sure they would bring relevant information to tests anyway.